packages/voqc/voqc.0.2.1/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "A verified optimizer for quantum circuits (VOQC)"
+description:
+  "OCaml library for VOQC, a Verified Optimizer for Quantum Circuits"
+maintainer: ["kesha@cs.umd.edu"]
+authors: ["Kesha Hietala"]
+license: "MIT"
+homepage: "https://github.com/inQWIRE/mlvoqc"
+doc: "https://inQWIRE.github.io/mlvoqc"
+bug-reports: "https://github.com/inQWIRE/mlvoqc/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.08"}
+  "openQASM"
+  "zarith"
```

Comment:
```suggestion
  "zarith" {>= "1.5"}
```

Reply:
For posterity: this package uses `Q.to_float` which was added in 1.5